packages/core/src/telemetry/metrics.test.ts

```diff
@@ -61,6 +61,13 @@ function originalOtelMockFactory() {
       setLogger: vi.fn(),
       warn: vi.fn(),
     },
+    DiagConsoleLogger: vi.fn().mockImplementation(() => ({})),
+    DiagLogLevel: {
+      INFO: 30,
+      DEBUG: 20,
+      WARN: 50,
+      ERROR: 60,
+    },
```

Comment:
i dont see this being used anywhere, am i missing something 🤔

Reply:
🤦‍♀️ 🤦‍♀️🤦‍♀️ thanks for the thorough explanation, I wasn't looking hard enough at the context this was in and thought it was an unused variable 🤪  - nothing to see here XD XD XD